crypto/cipher/cipher.go

```diff
@@ -56,5 +52,5 @@ type Decrypter interface {
 // Returned encrypted data must include what encryption method was used as the first byte.
 // The data can be decrypted using the corresponding PrivateKey and Decrypter method.
 type Encrypter interface {
-	Encrypt(pubKey crypto.PublicKey, plain PlainContent) (EncryptedContent, error)
+	Encrypt(PlainContent) (EncryptedContent, error)
```

Comment:
This is a simple but great change. I like how it makes the interface easier to understand and there is no current use case for using the same encrypter with different public keys.